src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -71,6 +71,24 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     assertExpectedRecords(WorkspaceAttributeRecord(dummyId2, workspaceId, "test", None, None, None, None, Option(-1), Option(0)))
   }
 
+  it should "save empty AttributeValueLists as AttributeEmptyList" in withEmptyTestDatabase {
+    val testAttribute = AttributeValueList(Seq())
+    runAndWait(workspaceQuery.save(workspace))
+    val numRows = workspaceAttributeQuery.insertAttributeRecords(workspaceId, "test", testAttribute, workspaceId).map(x => runAndWait(x))
+    assertResult(1) { numRows.head }
+
+    assertExpectedRecords(WorkspaceAttributeRecord(dummyId2, workspaceId, "test", None, None, None, None, Option(-1), Option(0)))
```

Comment:
It isn't a magic number - it's a dummy. The last parameter (= list length of zero) is what indicated an empty list.